stan/math/rev/core/vari.hpp

```diff
@@ -141,6 +148,12 @@ class vari {
   }
 };
 
+#ifndef STAN_THREADS
+ChainableStack& vari::ad_stack_ = ChainableStack::context();
```

Comment:
Why is this in a different section from the declaration?

Reply:
This is declaring a `static` global class member. I thought this has to be written like this, no?

Reply:
Oops, yes. only C++17 apparently allows inline initialization